homework/index.js

```diff
@@ -96,21 +90,23 @@
         const liElement = createAndAppend('li', ulElement, {
```

Comment:
To be exact, what you get back from the GitHub API is an array `contributors`. And an element of that array is a `contributor`. Therefore, the most descriptive and intention-revealing form for your `.forEach` would be:

```js
contributors.forEach(contributor => {
```

Names like `element` are generic and do not reveal much about what type of data they contain.